lib/Dialect/LWE/IR/NewLWEAttributes.td

```diff
@@ -271,30 +271,27 @@ def LWE_PlaintextSpaceAttr : AttrDef<LWE_Dialect, "PlaintextSpace"> {
 def LWE_KeyAttr : AttrDef<LWE_Dialect, "Key"> {
   let mnemonic = "key";
   let description = [{
-    An attribute describing the key used for encrypting the ciphertext.
-
-    This attribute includes a key identifier for the original key used to
-    encrypt the secret key.
-
-    The `key_size` parameter is used to describe the number of polynomials of
-    the secret key. This is typically $1$ for RLWE ciphertexts and greater than
-    $1$ for LWE instances. A ciphertext encrypted with a `key_size` of $k$ will
-    have size $k+1$.
-
-    The key basis describes the inner product used in the phase calculation in
-    decryption. This attribute is only supported for RLWE ciphertexts whose
-    `key_size` is $1$. An RLWE ciphertext is canonically encrypted against key
-    basis `(1, s)`. After a multiplication, its size will increase and the basis
-    will be `(1, s, s^2)`. The array that represents the key basis is
-    constructed by listing the powers of `s` at each position of the array. For
-    example, `(1, s, s^2)` corresponds to `[0, 1, 2]`, while `(1, s^2)`
-    corresponds to `[0, 2]`.
+    An attribute describing the key used for decrypting the ciphertext.
```

Comment:
I think it might be better to say "the key under which the message is currently encrypted" or something like that, as there'll be plenty of intermediate ciphertexts that are expressed relative to a key basis we don't actually ever intend to use for decryption.